ct-setting-item.html

```diff
@@ -126,7 +96,7 @@
           var value = input.value;
           var key = this.key;
           var category = this.category;
-          // switcher.editMode = !switcher.editMode;
+
           if(this._isStateEdit) {
             this.value = value;
             this.fire('edit', {"key" : this.key, "value": this.value, "category": this.category});
```

Comment:
Rename event to ct-save-setting-item-tapped. Always use the ct- prefix for our events and separate words with a dash -. Also keep event names in the past tense stating what happened